events/syscall/debugfs_launched_in_privilleged_container.go

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package syscall
+
+import (
+    "os/exec"
+    "syscall"
+
+    "github.com/falcosecurity/event-generator/events"
+)
+
+var _ = events.Register(LaunchDebugfsInPrivilegedContainer)
+
+func LaunchDebugfsInPrivilegedContainer(h events.Helper) error {
+    if h.InContainer() {
+        cmd := exec.Command("/usr/bin/debugfs", "/dev/sda1")
```

Comment:
```suggestion
        cmd := exec.Command("debugfs", "/dev/sda1")
```

Also, could we just use a tmp file instead of `/dev/sda1` (that looks scary :D) ?

Reply:
@leogr I checked on my system by launching privilege container with Dockerfile
```
FROM ubuntu:latest

RUN apt-get update && apt-get install -y e2fsprogs

USER root

CMD ["/bin/bash"]

```
and it triggered related rule. 
![Uploading Screenshot from 2024-03-28 02-01-19.png…]()

Reply:
and here we basically don't need to provide file location.